src/config/graphical-session/graphics-drivers/amd.md

```diff
@@ -0,0 +1,29 @@
+# AMD or ATI GPUs
+
+AMD GPU support requires the `linux-firmware-amd` package. If you have
+installed the `linux` or `linux-lts` packages, it will be installed as a
+dependency. If you installed a version-specific kernel package (e.g.,
+`linux5.4`), it may be necessary to manually install `linux-firmware-amd`.
+
+## OpenGL
+
+Install the Mesa DRI package, `mesa-dri`.
+
+> Note: This is already included in the `xorg` meta-package, but it is needed
+> when installing Xorg via `xorg-minimal` or for running a Wayland compositor.
```

Comment:
We are removing block quotes from the docs when they aren't quotes, so this should be a normal paragraph.